### ISSUE TITLE
Can you document how to pass date along with pincode also if possible allow all avaible dates to be fetched 

### ISSUE THREAD

**Issue by Shyamniper** [backtrackbaba/cowin#1]

**Comment by backtrackbaba** [comment]
Sure, I'll add it in the README on how you could pass the date along with the pin code.

As for all available dates to fetched, the API gives you by default get the data for the next 7 days (including today) with all the available slots on any given date.

Ex: When you pass the date as 04-05-2021, the response contains the slots up to 10-05-2021 as shown in the attached image.
![image](https://user-images.githubusercontent.com/9502343/116823338-8ab2f380-aba1-11eb-81f7-36e030932087.png)

So, basically, if you want to fetch for all the dates in the month of May starting from 4th May, you will simply need to query for the following dates:
4th May 2021
11th May 2021
18th May 2021
25th May 2021
and this should give you all the available slots for the month.

Hope this resolves your issue! Happy hacking!

**Comment by DragonWarrior15** [comment]
I think this is now resolved, we can probably close this.
